notifiers/github-app/cmd/main.go

```diff
@@ -60,9 +61,14 @@ func main() {
 		log.Fatalf("error creating github client: %v", err)
 	}
 
-	config, err := clientcmd.BuildConfigFromFlags("", filepath.Join(os.Getenv("HOME"), ".kube", "config"))
+	// creates the in-cluster config
+	config, err := rest.InClusterConfig()
 	if err != nil {
-		log.Fatalf("error getting kubernetes client config: %v", err)
+		log.Println("unable to find in-cluster kubeconfig. Falling back to local config")
+		config, err = clientcmd.BuildConfigFromFlags("", filepath.Join(os.Getenv("HOME"), ".kube", "config"))
```

Comment:
I think we can just use `BuildConfigFromFlags` . If no flags are specified, it defaults to rest.InClusterConfig

https://github.com/kubernetes/client-go/blob/v0.21.1/tools/clientcmd/client_config.go#L614-L616

Reply:
Done (though used knative libraries to keep to avoid flag conflicts).